core/src/libraries/Volatility.sol

```diff
@@ -80,21 +81,17 @@ library Volatility {
             // This is an approximation. Ideally the fees earned during each swap would be multiplied by the price
             // *at that swap*. But for prices simulated with GBM and swap sizes either normally or uniformly distributed,
             // the error you get from using geometric mean price is <1% even with high drift and volatility.
-            volumeGamma0Gamma1 = revenue1Gamma0 + amount0ToAmount1(revenue0Gamma1, data.arithmeticMeanTick);
-        }
+            uint256 volumeGamma0Gamma1 = revenue1Gamma0 + amount0ToAmount1(revenue0Gamma1, data.arithmeticMeanTick);
 
-        uint128 sqrtTickTVLX32 = uint128(
-            FixedPointMathLib.sqrt(
+            // Fits in uint128
+            uint256 sqrtTickTVLX32 = FixedPointMathLib.sqrt(
```

Comment:
I've seen auditors complain when using memory/stack data types other than `uint256`, because under the hood, all memory numbers are actually `uint256`. So even though it _would_ fit in `uint128`, the (imo silly) convention is to declare it as a `uint256`